### PR TITLE
Cooling and heating status support

### PR DESCRIPTION
**Pull request by jbicik**
Added support for the indication of cooling or heating
Fixed incorrect registry check

### CODE REVIEW

File: custom_components/atrea/climate.py [hunk 1]
```diff
@@ -256,6 +260,17 @@ def fan_modes(self):
...
+    @property
+    def hvac_action(self):
+        if (self._heating == 1):
+            return HVACAction.HEATING
+        elif (self._cooling == 1):
+            return HVACAction.COOLING
+        elif (self._current_hvac_mode == HVAC_MODE_OFF):
+            return HVACAction.OFF
+        else:
+            return HVACAction.IDLE
```
Comment (JurajNyiri) [inline custom_components/atrea/climate.py:272]: Wouldn't `return HVACAction.FAN` be better than `return HVACAction.IDLE`?

**Review by JurajNyiri — CHANGES_REQUESTED** [review]
Thank you for your contribution!

I tested this and everything seems to be working as expected. 
With the addition of `hvac_action` property the UI now changes, previously it was showing `Operation - Preset`, now it is showing `HVAC Action - Preset`, see screenshots below.

Before: 
image

After: 
image

I consider this to be worse experience for my unit, so I would like to suggest adding the property `hvac_action` dynamically, only if the `C10215` or `C10216` exists.

Could I ask you to do this changes or shall I?

File: custom_components/atrea/climate.py (hunk 1 above)
Comment (jbicik) [reply custom_components/atrea/climate.py]: You are right. Fan is better. Fixed in the latest commit. I was mainly after the cooling/heating as the history then shows the state nicely in the graph.

**Comment by jbicik** [comment]
Sorry, my Python knowledge is too limited and I have not managed to set the property dynamically. Do you think that you could try doing that?

**Comment by JurajNyiri** [comment]
Thank you, I will take a look 👍

**Comment by JurajNyiri** [comment]
Hey, I've just pushed an updated version, please let me know if it works for your unit.

**Comment by jbicik** [comment]
> Hey, I've just pushed an updated version, please let me know if it works for your unit.

Works well for me. Thank you.

**Pull request merged**